src/rules/no-nodejs-modules.js

```diff
@@ -2,7 +2,7 @@ import importType from '../core/importType'
 import isStaticRequire from '../core/staticRequire'
 
 function reportIfMissing(context, node, allowed, name) {
-  if (!allowed.includes(name) && importType(name, context) === 'builtin') {
+  if (allowed.indexOf(name) === -1 && importType(name, context) === 'builtin') {
```

Comment:
Not that surprising actually, since it's not even ES2015, but an ES2016 feature. I thought Babel would actually take care of this, but I guess was too hopeful (or it's just something to add). Anyway `indexOf` is fine.